clang/lib/CConv/ConstraintsGraph.cpp

```diff
@@ -50,27 +57,40 @@ void ConstraintsGraph::addConstraint(Geq *C, Constraints &CS) {
   }
   auto V1 = addVertex(A1);
   auto V2 = addVertex(A2);
+  add_edge(V2, V1, CG);
+}
 
-  EdgeType edgeType;
-  if (C->constraintIsChecked()) {
-    edgeType = Checked;
-  } else {
-    edgeType = Ptype;
-  }
 
-  add_edge(V2, V1, edgeType, CG);
+void GraphVizOutputGraph::mergeConstraintGraph(ConstraintsGraph Graph,
```

Comment:
I think that `Graph` should be a reference type?

Reply:
And: See comment about `getAllEdges` above; do something higher-order here instead.